File: batch_processing/parser.py
from model.change_event import ChangeEvent
class Parser():
    def __init__(self) -> None:
        pass

    def parse_change_event(self, event: ChangeEvent):
        pass

class DefaultParser(Parser):
    def __init__(self) -> None:
        super().__init__()

    def parse_change_event(self, event: ChangeEvent):
        return super().parse_change_event()

File: stream_processing/parser.go
package main

import (
	"stream_processing/core/logger"
	"stream_processing/model"
)

type DefaultParser struct {
}

func (p *DefaultParser) Parse(event *model.ChangeEvent) error {
	logger.WriteText(event.GetUniqueIdentity(), logger.INFO)
	return nil
}
